### ISSUE TITLE
Deploy k8gb on GCP/GKE

### ISSUE THREAD

**Issue by jkremser** [k8gb-io/k8gb#641]
Replicate our steps described in [local setup](https://www.k8gb.io/docs/local.html) on GCP or Google's managed K8s (gke). Including the
- `make test-failover`
- `make test-round-robin`
(the concepts captured by these Makefile targets)

Deliverables:
pull-request with the results describing the:
- reproduction steps (bash, makefile or markdown)
- issues that had to be solved in order to achieve this (platform specifics)
- all the steps that were necessary to make in the GCP web ui as well as the `gcloud ...` CLI invocations

NOTE: the files should land somewhere under `/docs/platforms/gcp/*`

You can use the free tier offering on [GCP](https://cloud.google.com/free/docs/gcp-free-tier).

If you need advice, we are on [Slack](https://cloud-native.slack.com/archives/C021P656HGB).

**Comment by nXtCyberNet** [comment]
Hi @ytsarev, can I take this? One clarification: the original issue mentions test-failover and test-round-robin, but k8gb has more functionalities now. Should I test all of them on GKE, or stick to those two?

**Comment by ytsarev** [comment]
@nXtCyberNet yes, feel free, I assigned this issue to you. You can start with main strategies first, the main goal is to test the k8gb deployment in specific environment end-to-end. Thank you!
